src/content/blog/tailscale-router-ddev-addon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "A Simple and Secure Way to Share Your DDEV Projects with Tailscale"
+pubDate: 2025-08-26
+summary: "Tired of temporary sharing links? Learn how to use the ddev-tailscale-router add-on to get a stable, secure, and private URL for your DDEV projects."
+author: Ajith Thampi Joseph
+featureImage:
+  src: /img/blog/2025/08/ddev_tailscale_flowchart.svg
+  alt: "DDEV Tailscale Router Cover Image"
+  caption: "The ddev-tailscale-router add-on enables cross-device testing, stable webhook URLs, and team collaboration."
+  credit: "Image by Claude AI :)"
+categories:
+  - Guides
+  - TechNotes
+---
+
+I've found that DDEV's `ddev share` command is a great way to quickly share my local development environment. However, since it uses ngrok, it generates a new, random URL every time, which can be a hassle. For a more permanent and secure solution, I've created the `ddev-tailscale-router` add-on.
```

Comment:
I would make a link to the add-on in the first paragraph so that people can go there right away.